modules/afsocket/afsocket-dest.c

```diff
@@ -207,6 +233,42 @@ afsocket_dd_start_reconnect_timer(AFSocketDestDriver *self)
   iv_timer_register(&self->reconnect_timer);
 }
 
+static gboolean
+_afsocket_dd_rename_connection_persist_name(AFSocketDestDriver *self, PersistState *state)
```

Comment:
If this called unconditionally, I'd consider renaming it to something that has a slight hint of conditionality in it.
`update_legacy_connection_persist_name()`, `rename_legacy_persist_name()`, etc.